tests/bson/bson-utcdatetime-004.phpt

```diff
@@ -18,10 +18,10 @@ foreach ($tests as $test) {
 --EXPECTF--
 object(MongoDB\BSON\UTCDateTime)#%d (%d) {
   ["milliseconds"]=>
-  int(%d)
+  string(13) "%d"
 }
 object(MongoDB\BSON\UTCDateTime)#%d (%d) {
   ["milliseconds"]=>
-  int(%d)
+  string(13) "%d"
```

Comment:
Consider using `%d` for the string length for all places where you also accept a `%d` in the pattern.